docs/api-reference/next/image.md

```diff
@@ -64,7 +64,12 @@ The `<Image />` component requires the following properties.
 
 ### src
 
-The path or URL to the source image. This is required.
+Can be either:
+1) A statically imported image file, as in the example code above, or
+2) A path to a remotely hosted image. This can be either an absolute URL,
+  or a relative path to be used with a [loader.](/docs/basic-features/image-optimization.md#loader)
+
+This is required.
```

Comment:
```suggestion
Required and must be one of the following:
1. A statically imported image file, as in the example code above, or
2. A path string. This can be either an absolute URL,
  or a relative path depending on the [loader](#loader).
```